fail2ban/server/server.py

```diff
@@ -324,6 +324,15 @@ def setUnbanIP(self, name, value):
 	def getBanTime(self, name):
 		return self.__jails[name].actions.getBanTime()
 	
+	def isAlive(self, jailnum=None):
+		if jailnum is not None and len(self.__jails) != jailnum:
+			return 0
```

Comment:
what is the purpose of haing jailnum argument here?  why it must be equal to number of jails???  why having Server.isAlive output must be integer (0, 1) and not bool, and whenever some jailnum is equal to number of jails???  very confused.  I guess docstring could help to understand... may be it is more of "areAllJailsAlive"? ;-)